recipes/yacman/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "yacman" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 37cd49463880f4c9cfee0dc8280bc893fc3bf392615d12b35c97609def71071c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - attmap >=0.12.9
+    - oyaml
+    - python
+    - pyyaml >=3.13
+    - ubiquerg >=0.4.9
```

Comment:
This package does not exist in ``conda-forge``, you will need to package ``ubiquerg`` and ``attmap`` first

Reply:
PRs are opened for these

Reply:
both exist in `conda-forge` now